readme.md

```diff
@@ -19,13 +19,13 @@ Copy moxConfig.js to your project test folder. Put mox.js and moxConfig.js in yo
       'test/moxConfig.js',
     ...
     ],
-    
+
```

Comment:
Trailing whitespace zou eigenlijk niet uit een markdown-bestand weggehaald mogen worden. In dit geval is het niet erg, maar er schijnen gevallen ze zijn waarbij whitespace relevant is voor de output.
Ik zit net readme.md aan te passen dus ik krijg straks wel een dik merge conflict :(